python/test/cli/test_makevarmap_Linux_LE_x64_Gcc_8_3_0.py

```diff
@@ -14,7 +14,7 @@
 
 from scrutiny.core import *
 from scrutiny.core.bintools.elf_dwarf_var_extractor import ElfDwarfVarExtractor
-from scrutiny.core.memdump import Memdump
+from scrutiny.core.memcontent import MemContent
```

Comment:
Careful about abbreviations for class names. `MemoryContent` would be a better name IMHO

Reply:
MemoryContent it is